src/rules/store/prefer-action-creator-in-dispatch.ts

```diff
@@ -0,0 +1,43 @@
+import type { TSESTree } from '@typescript-eslint/experimental-utils'
+import { ESLintUtils } from '@typescript-eslint/experimental-utils'
+import path from 'path'
+import { docsUrl, findNgRxStoreName, storeDispatch } from '../../utils'
+
+export const messageId = 'preferActionCreatorInDispatch'
+export type MessageIds = typeof messageId
+
+type Options = []
+
+export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
+  name: path.parse(__filename).name,
+  meta: {
+    type: 'problem',
```

Comment:
Should we use `suggestion` for all of our `prefer-` rules 🤔

Reply:
Addressed 💯. Your comment made me realize that today we have `prefer-concat-latest-from` as `type: 'suggestion'`, but shouldn't it be `problem` instead, as misusing `withLatestFrom` can lead to some problems?

Reply:
You're also right! That should be `problem`.
This is hard 😅